### ISSUE TITLE
Altair spec tests for ssz and mainnet

### ISSUE THREAD

**Issue by twoeths** [ChainSafe/lodestar#2580]
1. `minimal` spec test of altair was added in #2554, need to add respective `mainnet` test
2. ssz spec test for altair was not added
+ It works fine with `SyncCommittee`
+ But failed with `SyncAggregate`
```
 70) SSZ - SyncAggregate ssz_random minimal
       case_15:
     Error: Invalid deserialized bitvector length
      at BitVectorType.tree_deserializeFromBytes (/Users/tuyennguyen/Projects/workshop/lodestar/node_modules/@chainsafe/ssz/src/types/composite/bitVector.ts:124:13)
      at ContainerType.tree_deserializeFromBytes (/Users/tuyennguyen/Projects/workshop/lodestar/node_modules/@chainsafe/ssz/src/types/composite/container.ts:369:21)
      at ContainerType.tree_deserialize (/Users/tuyennguyen/Projects/workshop/lodestar/node_modules/@chainsafe/ssz/src/types/composite/abstract.ts:119:17)
      at ContainerType.createTreeBackedFromBytes (/Users/tuyennguyen/Projects/workshop/lodestar/node_modules/@chainsafe/ssz/src/types/composite/abstract.ts:396:39)
      at Object.expectFunc (test/spec/ssz/util/testCases.ts:83:32)
      at Context.<anonymous> (/Users/tuyennguyen/Projects/workshop/lodestar/packages/spec-test-util/src/single.ts:142:15)
```
+ Need to add ssz tests for remaining types in `spec-tests/tests/minimal/altair/ssz_static`

**Comment by dapplion** [comment]
Issue is fixed

**Comment by twoeths** [comment]
got next issue
```
30) SSZ - SyncCommitteeContribution ssz_random minimal
       case_9:
     Error: Data length 145 is too small, expect at least 149
```

**Comment by twoeths** [comment]
issue fixed, waiting for ssz v0.8.8 to pass spec tests
